File: src/test/console/php/services/KeyService.php
<?php

function getAPIKey() {
  include("configs.php");

  $url = $testServerUrl . "/api-keys";
  $ch = curl_init($url);

  $headers = array(
    "token: ".$testServerSecretKey
  );
  curl_setopt($ch, CURLOPT_HTTPHEADER, $headers);
  curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
  $body = curl_exec($ch);

  curl_close($ch);
  $allKeys = json_decode($body);

  $key = $allKeys[0];
  return $key;
}

File: src/test/console/python/services/KeyService.py
import requests
import sys
sys.path.append('../..')
from python import configs

class KeyService:
  def getApiKey(self):
    hostUrl = configs.testServerUrl
    headers = {'token': configs.testServerSecretKey}
    rq = requests.get(hostUrl + '/api-keys', headers = headers)

    if rq.status_code == 200:
      key = rq.json()[0]
    else:
      key = None

    return key
